natiami/assignment4/FindIslands.h

```diff
@@ -0,0 +1,16 @@
+#include<vector>
+using namespace std;
+
+class FindIslands{
+
+public:
+	int nr_row;
+	int nr_column;
+	vector<vector<bool> > mapOfTiles;

```

Comment:
Tip:
You can replace `mapOfTiles` with a name that better represents the meaning of the content of the map.
What "true" and "false" mean here? True means land, False mean sea.
So `isLand` or `landTiles` or `isLandTile` can make the code read more like English:
```
if (isLandTile[i][j]) {  // "if (i,j) is a land tile then... "
   ...
}
```